Dshop/apps/products_catalogue/templates/products_catalogue/products_list.html

```diff
@@ -19,9 +21,17 @@ <h5 class="card-title"><a href="{% url 'product-detail' object.slug object.pk %}
                     <p>Brak zdjęć</p>
                     {% else %}
                         {% for pic in object.featured_photos %}
-                        <a href="{% url 'product-detail' object.slug object.pk %}"><img src="{{ pic.image.url }}" alt="{{ object.name }}"/></a>
+                            {% thumbnail pic.image "200x200" crop="center" as im %}
+                                <a href="{% url 'product-detail' object.slug object.pk %}">
+                                    <img src="{{ pic.image.url }}" width="{{ im.width }}" height="{{ im.height }}">
+                                </a>
+                            {% endthumbnail %}
                         {% empty %}
-                        <a href="{% url 'product-detail' object.slug object.pk %}"><img src="{{ object.images.first.image.url }}" alt="{{ object.name }}"/></a>
+                            {% thumbnail object.images.first.image "200x200" crop="center" as thumb %}
+                                <a href="{% url 'product-detail' object.slug object.pk %}">
+                                    <img src="{{ object.images.first.image.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}">
```

Comment:
This is not correct. 
object.images.first.image.url  is not thumbnail, it's original, huge image. Please use thumbnail there - this should be `thumb.url`
https://github.com/jazzband/sorl-thumbnail#templates-usage